src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -518,6 +518,37 @@ ExportQueryExecutionTrees::idToLiteral(const Index& index, Id id,
   }
 }
 
+// _____________________________________________________________________________
+std::optional<ad_utility::triple_component::LiteralOrIri>
+ExportQueryExecutionTrees::idToLiteralOrIri(const Index& index, Id id,
+                                            const LocalVocab& localVocab) {
+  using enum Datatype;
+  switch (id.getDatatype()) {
+    case WordVocabIndex:
+      return LiteralOrIri{
+          ad_utility::triple_component::Literal::literalWithoutQuotes(
+              index.indexToString(id.getWordVocabIndex()))};
```

Comment:
I think this pattern is also repeated in this file, please factor out the function `getLiteralOrIriFromWordVocabIndex`